src/session/get-access-token.ts

```diff
@@ -129,23 +129,24 @@ export default function accessTokenFactory(
       }
     }
 
-    // Check if the token has expired.
-    // There is an edge case where we might have some clock skew where our code assumes the token is still valid.
-    // Adding a skew of 1 minute to compensate.
-    if (!session.refreshToken && session.accessTokenExpiresAt * 1000 - 60000 < Date.now()) {
-      throw new AccessTokenError(
-        'access_token_expired',
-        'The access token expired and a refresh token is not available. The user will need to sign in again.'
-      );
-    }
 
     // Check if the token has expired.
     // There is an edge case where we might have some clock skew where our code assumes the token is still valid.
     // Adding a skew of 1 minute to compensate.
     if (
-      (session.refreshToken && session.accessTokenExpiresAt * 1000 - 60000 < Date.now()) ||
-      (session.refreshToken && accessTokenRequest && accessTokenRequest.refresh)
+      (session.accessTokenExpiresAt * 1000 - 60000 < Date.now()) ||
+      (accessTokenRequest && accessTokenRequest.refresh)
     ) {
+      // Check if the token has expired.
+      // There is an edge case where we might have some clock skew where our code assumes the token is still valid.
+      // Adding a skew of 1 minute to compensate.
+      if (!session.refreshToken) {
```

Comment:
Also, for readability, rather than nesting these if branches, can you create a top level one, along the lines of 

```js
if (!session.refreshToken && accessTokenRequest?.refresh) {
  throw new AccessTokenError(...)
}
```